client/src/components/admin/ResultAttemptsForm/ResultAttemptsForm.jsx

```diff
@@ -106,6 +106,7 @@ function ResultAttemptsForm({
       return confirm({
         description: submissionWarning,
         confirmationText: "Submit",
+        confirmationKeyword,
```

Comment:
```suggestion
        description: submissionWarning.text,
        confirmationText: "Submit",
        confirmationKeyword: submissionWarning.confirmationKeyword,
```

Reply:
just noting that I used `description` instead of `text` so that it can just be
```
{
  description,
  confimationText: "Submit",
  confirmationKeyword,
}
```

by destructuring on line 99.  Then the conditional checking for `submissionWarning` can be changed to `description` since that is analogous to the previous functionality.